napari_clusters_plotter/_dimension_reduction.py

```diff
@@ -0,0 +1,238 @@
+import pandas as pd
+import warnings
+import napari
+from qtpy.QtWidgets import QWidget, QPushButton, QLabel, QHBoxLayout, QVBoxLayout
+from qtpy.QtWidgets import QListWidget, QListWidgetItem, QAbstractItemView, QComboBox, QSpinBox
+from qtpy.QtCore import QRect
+from ._utilities import widgets_inactive
+from napari_tools_menu import  register_dock_widget
+
+@register_dock_widget(menu="Measurement > Dimensionality reduction (ncp)")
+
+class dimredWidget(QWidget):
```

Comment:
```suggestion
class DimensionalityReductionWidget(QWidget):
```
Please use proper classnames. Also Python classes start upper case.